test/server.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const accounts = require('../index.js');
+
+const chai = require('chai');
+const chai_as_promised = require('chai-as-promised');
+const nock = require('nock');
+const httpMocks = require('node-mocks-http');
+
+
+chai.should();
+chai.use(chai_as_promised);
+
+const configuration = {
+    base_url: 'https://accounts.freedom.tm',
```

Comment:
If this is going to be a public library, let's not use our application name/url/scopes and keep it generic